clients/cloud/confluent-cli/confluent-cli-example.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+topic_name=test1
+
+ccloud topic create $topic_name
+
+num_messages=10
+for ((i=0;i<$num_messages;i++)); do
+  echo "alice,{\"count\":${i}}" | confluent produce $topic_name --cloud --property parse.key=true --property key.separator=,
+done
+
+confluent consume $topic_name --cloud --property print.key=true --from-beginning --max-messages $num_messages
```

Comment:
If the example is re-run it will still only show the messages from the first run.
Suggest removing --max-messages, user can always ctrl-c.